lib/Doctrine/Common/Persistence/AbstractManagerRegistry.php

```diff
@@ -190,7 +190,10 @@ public function getManagerForClass($class)
     {
         // Check for namespace alias
         if (strpos($class, ':') !== false) {
-            list($namespaceAlias, $simpleClassName) = explode(':', $class);
+            if (2 < count($classNameAlias = explode(':', $class))) {
```

Comment:
Note that `explode()` has a third `$limit` argument (see http://3v4l.org/as2I7), maybe that's a better solution here...